src/attributes/blob/server/index.ts

```
@@ -88,4 +87,49 @@ IsAttributeStorage
       new BlobChange(new Blob(), insertResult.id),
     );
   }
+
+  private async marshal(value: Blob): Promise<string> {
+    return `data:${value.type};base64,${Buffer.from(await value.arrayBuffer()).toString('base64')}`;
+
+    // TODO: this does not work with the current implementation of S3Storage
+    return this.isS3Configured
+      ? value.text()
+      : `data:${value.type};base64,${Buffer.from(await value.arrayBuffer()).toString('base64')}`;
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)